components/decimal/README.md

```diff
@@ -24,7 +25,7 @@ let fdf = FixedDecimalFormatter::try_new_unstable(
 )
 .expect("Data should load successfully");
 
-let fixed_decimal = 1000007.into();
+let fixed_decimal = FixedDecimal::from(1000007);
```

Comment:
issue: as mentioned in the main issue I'm not convinced this is a problem, `.into()` is standard rust practice.

I think explicitly mentioning the type may still be useful from the pov of examples, though I'm not sure it matters here since the variable is named appropriately.

(a side effect I dislike about explicitly spelled out examples is that people copy them into their code without cleaning them up)

Reply:
followup from the discussion there: still not fully happy with it but shane's comment convinces me that there's decent reason to do it here